README.md

```diff
@@ -1,5 +1,97 @@
+# Awesome Semantic Shapes [![Awesome](https://awesome.re/badge.svg)](https://awesome.re)
 
-# Specification 'awesome-semantic-shapes'
+> A curated list of Semantic Shapes resources. Contributions welcome! Please, read the [Contribution Guidelines](CONTRIBUTING.md) first.
 
-This is the repository for awesome-semantic-shapes. You're welcome to contribute! Let's make the Web rock our socks
-off!
+Semantic shapes enables you to validate RDF graphs against a set of conditions. Semantic shapes can also be viewed as a description of the data graphs that do satisfy these conditions. Such descriptions may be used for a variety of purposes beside validation, including user interface building, code generation and data integration. Semantic shapes could be described in SHACL or ShEx languages.
+
+## Contents
+
+- [SHACL Validators](#shacl-validators)
+- [ShEx Validators](#shex-validators)
+- [Shapes Discovery Tools](#shapes-discovery-tools)
+- [Shapes Collections](#shapes-collections)
+- [Shape Conversion Tools](#shape-conversion-tools)
+- [Shape Generators](#shape-generators)
+- [Shape Editors, Visualizations](#shape-editors-visualizations)
+- [Declarative UIs](#declarative-uis)
+- [Book](#book)
+- [Tutorials](#tutorials)
+- [Presentations](#presentations)
+- [Specifications](#specifications)
+- [Legend](#legend)
+
+## SHACL Validators
+
+- [Jena SHACL](https://github.com/apache/jena/) -- Java, Supports: SHACL Core, SHACL-SPARQL
```

Comment:
We want the link to be most specific, into the SHACL documentation of the concrete product

Reply:
Do I understand correctly that all "validator links" in these two sections should point to the (list in a descending priority):

1. SHACL/ShEx documentation of this validator (for Jena or RDF4J, or Corese), and no need to add link to the sources repository in a source control system?
2. If no validator documentation was found, then the link should point to the validator module folder in the sources repository in a source control system?

Reply:
> * Afaik, English grammar uses `: ` not `--`

I checked with the [awesome-lint](https://github.com/sindresorhus/awesome-lint), it should be one dash `-` between a link and a description, not em-dash, not double-dash '--'. And it should be a comma in the end if there is a description (or no comma if a link only).

This simple construction -- is the most often one

[software] - Description

Or it could be a more complex construction (for example, here [brillout/awesome-react-components](https://github.com/brillout/awesome-react-components))

[software] - [docs] - [demo] - Description

Reply:
Using a colon (without preceding space, with following space) instead of the em-dash is valid. There should probably be a period (not a comma) following the language in which the thing was written. There's no need for the colon following "Supports". There should probably be a period at the end of the line.
```
- [Jena SHACL](https://github.com/apache/jena/): Java. Supports SHACL Core, SHACL-SPARQL.
```

The "more complex construction" is yet more ugliness, to my eyes. I would much prefer a proper table, where any cell's text could wrap within the cell. The presented format is useful where tables are hard to write and/or render — but that's not the case for any of this content.

I don't know why [awesome-lint](https://github.com/sindresorhus/awesome-lint) should be treated as an authority on human-facing punctuation. I'm sure it does a pretty good job on linting *code*, but Markdown is, by design, closer to human language than to programming. I know coders tend to prefer hyphens, because they don't require a key-chord or more complex entry method, but hyphens aren't appropriate for the placements described.

If the reason the not-so-[awesome-lint](https://github.com/sindresorhus/awesome-lint) is being granted such authority is that this is an "awesome"-list, well, I think the "awe" it will trigger will be less than the "eww", and I suggest reconsideration and pushing back upstream.

Reply:
[awesome-lint](https://github.com/sindresorhus/awesome-lint) is a specialized markdown linter, tailored specifically to lint "awesome-XXX" lists of some "awesome stuff", created by the author of "awesome list of awesome lists". Some ad-hoc domain-specific convention.